transforms/.make.cicd.targets

```diff
@@ -28,62 +32,113 @@ clean:: .transforms.clean
 set-versions::
 
 ## We need to think how we want to do this going forward
-build::
 
-image::
-	@if [ -e Dockerfile ]; then \
-		$(MAKE) image-default ; \
-	else \
-		echo "Skipping image for $(shell pwd) since no Dockerfile is present"; \
+build:: image
+
+publish:
+	@if [ -e Dockerfile.python ]; then \
+		$(MAKE) DOCKER_REMOTE_IMAGE=$(DOCKER_REGISTRY_ENDPOINT)/$(TRANSFORM_NAME)-python:$(DOCKER_IMAGE_VERSION) \
```

Comment:
Instead of setting DOCKER_REMOTE_IMAGE, which is delicate, I would use DOCKER_IMAGE_NAME and let DOCKER_REMOTE_IMAGE be set from that in .make.defaults.  This way it seems less delicate to me.

Reply:
In fact, you've done this below for `test-image` and others.